deploy/provision/glances-autodetect.ps1

```
@@ -0,0 +1,490 @@
+
+# ---------------------------------------------------------------------------
+# Platform hints
+# ---------------------------------------------------------------------------
+function Get-PlatformHints {
+    $hasTailscale = $null -ne (Get-Service -Name 'Tailscale' -ErrorAction SilentlyContinue) -or
+                   (Test-Path 'C:\Program Files\Tailscale\tailscale.exe')
+    $hasDocker    = $null -ne (Get-Service -Name 'com.docker.service' -ErrorAction SilentlyContinue) -or
+                   (Test-Path 'C:\Program Files\Docker\Docker\Docker Desktop.exe')
+    $hasHyperV    = (Get-Service -Name 'vmms' -ErrorAction SilentlyContinue).Status -eq 'Running'
+    $hasWsl2      = $null -ne (Get-Command 'wsl' -ErrorAction SilentlyContinue) -and
+                   ((wsl --list --quiet 2>$null) -ne $null)
```

PSScriptAnalyzer (1.25.0)

[warning] $null should be on the left side of equality comparisons.

Suggested fix: Use $null on the left hand side for safe comparison with $null.

(PSPossibleIncorrectComparisonWithNull)